pkg/utils/typeutil/size_test.go

```diff
@@ -39,24 +39,32 @@ func TestSizeJSON(t *testing.T) {
 }
 
 func TestParseMbFromText(t *testing.T) {
```

Comment:
If don't use this fix, the new test can still work?

Reply:
The fix doesn't involve any change to `ParseMBFromText`.

Reply:
got, Do we need to test this fix?